backend/src/test/java/com/woowacourse/kkogkkog/application/AuthServiceTest.java

```diff
@@ -0,0 +1,36 @@
+package com.woowacourse.kkogkkog.application;
+
+import com.woowacourse.kkogkkog.application.dto.TokenResponse;
+import com.woowacourse.kkogkkog.domain.Member;
+import com.woowacourse.kkogkkog.domain.repository.MemberRepository;
+import com.woowacourse.kkogkkog.fixture.MemberFixture;
+import com.woowacourse.kkogkkog.presentation.dto.TokenRequest;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest
+@Transactional
+class AuthServiceTest {
+
+    @Autowired
+    private MemberRepository memberRepository;
+
+    @Autowired
+    private AuthService authService;
+
+    @Test
+    @DisplayName("로그인에 성공하면 토큰을 반환한다")
+    void login() {
+        Member rookie = MemberFixture.ROOKIE;
+        memberRepository.save(rookie);
+
+        TokenRequest tokenRequest = new TokenRequest(rookie.getEmail(), rookie.getPassword());
```

Comment:
```suggestion
        memberRepository.save(ROOKIE);

        TokenRequest tokenRequest = new TokenRequest(ROOKIE.getEmail(), ROOKIE.getPassword());
```
픽스쳐 등록되어 있는건 바로 사용하는건 어떤가요??

Reply:
`ROOKIE`라고 static import를 하지 않은 이유가 혹시 있나요?

Reply:
딱히 이유는 없었습니다! 
반영했습니다

Reply:
픽스쳐를 바로 사용하지 않은 이유는 밑의 `TokenRequest`와의 일관성을 유지해주기 위해서입니다. 서로 다른 파라미터로 관리하게 되면 휴면이슈 가능성이 있기 때문에 `Fixture`를 받아서 하는 설계를 선호합니다. 

static import는 누락된 부분이네요 추가로 반영했습니다 👍